src/internal/components/QrCode/gradientConstants.ts

```diff
@@ -0,0 +1,65 @@
+export const QR_CODE_SIZE = 237;
+export const QR_LOGO_SIZE = 50;
+export const QR_LOGO_RADIUS = 10;
+export const QR_LOGO_BACKGROUND_COLOR = '#ffffff';
+export const GRADIENT_START_COORDINATES = { x: 0, y: 0 };
+export const GRADIENT_END_COORDINATES = { x: 1, y: 0 };
+export const GRADIENT_END_STYLE = { borderRadius: 32 };
+
+type LinearGradient = { startColor: string; endColor: string };
+
+export const ockThemeToLinearGradientColorMap = {
+  default: 'blue',
+  base: 'baseBlue',
+  cyberpunk: 'pink',
+  hacker: 'black',
+};
+
+export const ockThemeToRadiamGradientColorMap = {
+  default: 'default',
+  base: 'blue',
+  cyberpunk: 'magenta',
+  hacker: 'black',
+};
+
+export const linearGradientStops: Record<string, LinearGradient> = {
+  blue: {
+    startColor: '#266EFF',
+    endColor: '#45E1E5',
+  },
+  pink: {
+    startColor: '#EE5A67',
+    endColor: '#CE46BD',
+  },
+  black: {
+    startColor: '#a1a1aa',
+    endColor: '#27272a',
+  },
+  baseBlue: {
+    startColor: '#0052ff',
+    endColor: '#b2cbff',
+  },
+};
```

Comment:
is it possible to make all of these pull from theme colors so when a user customizes a theme (or adds their own) the qr code adapts?

Reply:
Agreed - we should avoid having to update this file when adding new themes. 

Has the QR component been tested with custom themes? Curious if it's updating as expected.